server/settings.go

```diff
@@ -14,14 +14,15 @@ const (
 
 	errStoreNewSettings = "Could not store new settings. Please contact your system administrator. Error: %v"
 	errConnectToJira    = "Your account is not connected to Jira. Please type `/jira connect`. %v"
-	subCommandAssignee  = "assignee"
-	subCommandMention   = "mention"
-	subCommandReporter  = "reporter"
-	subCommandWatching  = "watching"
+
+	notificationRoleAssignee = "assignee"
+	notificationRoleMention  = "mention"
```

Comment:
you have defined same strings as constant twice with different variable give it a generic name initialise it only once